<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>238</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>491</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>配置表文件路径</string>
   </property>
   <widget class="QLineEdit" name="textScenePath">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>320</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelNpcFilePath">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>40</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>Npc.xlsx 路径</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlotFilePath">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>80</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>Plot.xlsx 路径</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="textNpcPath">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>320</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFindScene">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSceneFilePath">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>120</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>Scene.xlsx 路径</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFindPlot">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFindNpc">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textPlotPath">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>320</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMissionFilePath">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>160</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="text">
     <string>Quest.xlsx 路径</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="textMissionPath">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>320</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFindMission">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>28</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
